### DISCUSSION TITLE
Httpx crash

### DISCUSSION THREAD

**Discussion by root-Manas** [projectdiscovery/httpx#1507] [Q&A]
I don't know what it is but, there are 2 packages named httpx on my machine one is httpx python library and other is httpx by pdtm whenever i try to run it httpx defaults to python library so I decided to remove it and then use httpx by pdtm but cant as pip says install httpx how do I fix this?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
